app/src/test/java/org/mozilla/fenix/components/metrics/LeanplumMetricsServiceTest.kt

```diff
@@ -31,9 +31,11 @@ class LeanplumMetricsServiceTest {
         assertNull(sharedPreferences.getString("LP_DEVICE_ID", null))
 
         val leanplumMetricService = LeanplumMetricsService(testContext.application, idGenerator)
+        leanplumMetricService.start()
```

Comment:
nit: maybe add a comment here and below that this is needed to init deviceId?

It's curious that in the original implementation, a deviceId could be fetched even if the user had telemetry disabled, huh? StrictMode seems to be making things better in other ways too. :)